dts/bindings/ethernet/atmel,sam-gmac.yaml

```diff
@@ -41,6 +41,11 @@ properties:
           - "mii"
         default: "rmii"
 
+    peripheral-id:
+      type: int
+      description: Peripheral ID
```

Comment:
would be nice to note here that `peripheral-id` is only applicable to the SAM (and not SAM0) family devices.

Reply:
SAM0 is a different SoC family. Nothing in this file is applicable to SAM0 devices, not only `peripheral-id` property. I would find such comment confusing.

Reply:
@mnkp GMAC is available in SAM E5x, which is SAM0 family.

Reply:
Sorry, I missed this. @stephanosio Your comment is valid.

Reply:
Add a default value to peripheral-id. With that, all SAM0 families will be OK. Besides peripheral-id be mandatory for any SAM, I think this is best relation between devicetree standards vs shared functionality. Otherwise, it will be odd have required: true and a default: 0 or peripheral-id at SAM0 dtsi.